doc/OVERMAP.md

```diff
@@ -245,6 +245,8 @@ rotation for the referenced overmap terrains (e.g. the `_north` version for all)
 | `mapgen_tee`      | Specify a C++ mapgen function for a LINEAR feature variation. Prefer JSON instead.               |
 | `mapgen_four_way` | Specify a C++ mapgen function for a LINEAR feature variation. Prefer JSON instead.               |
 | `eoc`             | Supply an effect_on_condition id or an inline effect_on_condition.  The condition of the eoc will be tested to see if the special can be placed.  The effect of the eoc will be run when the special is placed.  See [effect_on_condition.md](effect_on_condition.md). |
+| `entry_eoc`       | An effect on condition ID that will run when you enter this location.                            |
+| `exit_eoc`        | An effect on condition ID that will run when you exit this location.                            |
```

Comment:
Small oopsies, but it looks like the case doesn't match between the docs and the strings loaded in `assign` ("entry_eoc" vs. "entry_EOC")

Reply:
good catch